infra/test/infra_test.go

```diff
@@ -132,13 +132,23 @@ func EnableDestroyService(t *testing.T, terraformOptions *terraform.Options) {
 		},
 		WorkingDir: "../../",
 	})
+	shell.RunCommand(t, shell.Command{
```

Comment:
What is this section for?

Reply:
By default, deletion protection is on for cognito. This script is for CI testing the infra service layer. To be able to delete cognito, we have to explicitly disable deletion protection.